facebook/react-native
# <Text> Expose Android's includeFontPadding property to JavaScript.

By default Android will put extra space above text to allow for upper-case accents or other ascenders. With some fonts, this can make text look slightly misaligned when centered vertically.

We have found that the effect is very noticeable with certain custom fonts on Android. On iOS the font aligns vertically as expected.

Android exposes a property `includeFontPadding` that will remove this extra padding if set to false. This PR exposes that to JS, and adds it to the documentation and UIExplorer.

Test Plan:
- Run UIExplorer
- Scroll down to find the <Text> option
- Click and scroll to bottom to see a sample with the default text vertical alignment (includeTextPadding: true) and the result when using includeTextPadding: false.

![screen shot 2016-08-09 at 17 38 05](https://cloud.githubusercontent.com/assets/288569/17527131/066b77b4-5e62-11e6-937a-fc30953cc469.png)
